<!DOCTYPE html>
<html class="no-js" lang="en">
<head><title>Submissions Overview &lt; ArrayExpress &lt; EMBL-EBI</title></head>
<body class="level2">
    <div id="content" role="main" class="grid_24 clearfix">
        <section id="ae-help" class="grid_18 alpha">
            <h2>How to submit</h2>
            
            <div class="grid_12">
                <div class="calloutbox" style="padding: 20px; position: relative; background-color: rgb(238, 249, 243);">
                    <h3 class="icon icon-functional" data-icon="_"><a href="/cgi-bin/microarray/magetab.cgi">MAGE-TAB spreadsheet tool</a></h3>
                    <br>
                    <ul>
                        <li>All sequencing and microarray experiment submissions</li>
                        <li><a href="/fgpt/magetab/help">MAGE-TAB submission help</a></li>
                        <li><a href="/training/online/course/arrayexpress-submitting-data-using-mage-tab">MAGE-TAB submission interactive tutorial</a></li>    
                    </ul>
                </div>
            </div>
            <div class="grid_12">
                <div class="calloutbox" style="padding: 20px;  position: relative; background-color: rgb(238, 249, 243);">
                    <h3 class="icon icon-functional" data-icon="_"><a href="/fg/annotare/login">Annotare webform tool</a></h3>
                    <br>
                    <ul>
                        <li>Intended for experiment submissions with 15 samples or fewer and no complex design (e.g. pooling of samples). </li>
                        <li>Larger/complex submissions will be supported soon.</li>
                        <li><a href="/fgpt/annotare_help">Annotare help</a></li>
                        
                        
                    </ul>
                </div>
            </div>

            <p>&nbsp;</p>
            <p>&nbsp;</p>

            
            <h3>More information about submissions</h3>
            <p>
                <a href="#WhatIsASub">1. What do I need to submit?</a><br>
                <a href="#ExptSubTools">2. Which experiment submission tool to use?</a><br>
                <a href="#WhatToExpect">3. What to expect from each experiment submisssion tool</a><br>
                <a href="#CommonQuestions">4. Common questions</a><br>
                <a href="#ADFSub">5. Array Design submissions</a><br>
            </p>
            <h3><a id="WhatIsASub"></a>1. What do I need to submit?</h3>
            <p>ArrayExpress accepts all functional genomics data, including microarray-based and <a href="UHTS_submissions.html">sequencing</a> data. Microarray submissions follow the "Minimum Information About a Microarray Experiment" (<a href='http://www.mged.org/Workgroups/MIAME/miame_2.0.html'>MIAME</a>) guidelines, while sequencing submissions follow a similar set of guidelines, "Minimum Information About a Sequencing Experiment" (<a href='http://www.mged.org/minseqe/'>MINSEQE</a>).  Following the guidelines, for experiment submissions, we ask submitters to provide the following:</p>
            <div class="grid_1 alpha">
                <p>&nbsp;</p>
            </div>

            <div  class="grid_17 omega">
                <p><strong>Meta-data</strong>: E.g. Background biology of your experiment, aim of the experiment, the biological materials/samples used and their characteristics, wet-lab and dry-lab procedures (protocols).</p>
                <p><strong>Raw data files</strong>: These are unprocessed data files obtained from the microarray scanner (e.g. Affymetrix CEL files, Agilent feature extraction files) or from the sequencing machine (e.g. fastq files).</p>
                <p><i>Microarray experiment submitters</i>: Please make sure the format of your raw data files matches one of the <a href="/fgpt/magetab/help/data_format.html#raw">accepted formats</a>.  For Illumina BeadChip microarray experiments, make sure you generated raw data files using the <a href="http://www.illumina.com/software/genomestudio_software.ilmn">Illumina GenomeStudio software </a>(previously known as the "BeadStudio" software) with the 'ArrayExpress Data Submission Report Plug-in'. More about <a href="faq.html#illumina">Illumina BeadChip submission</a>.</p>
                <p><i>Sequencing experiment submitters</i>: Please make sure your raw sequence read files are formatted and prepared according to <a href="/ena/about/read-file-formats">specifications by the European Nucleotide Archive (ENA)</a> and you've gathered all required information describing your samples/libraries/data files. See this <a href = "UHTS_submissions.html">specific help page on sequencing submissions</a> for more information.</p>
                <p><strong>Processed data files</strong>: These are data files generated from the raw data files, often involving e.g. normalisation or background-subtraction. In sequencing expermient, the processing can also involve alignment of sequencing reads to a reference genome, calculation of RPKM/FPKM values, etc.</p>
                <p><i>Microarray experiment submitters</i>: Please prepare files as in this <a href="/fgpt/magetab/help/data_format.html#normalized">per-hybridisation normalised file example</a> or the <a href="/fgpt/magetab/help/data_format.html#data_matrix">data matrix format</a>. We do not enforce strict formatting requirements for processed data files, but you're encouraged to use the recommended formats.  Please also make sure you have provided a detailed normalisation/data transformation protocol in your meta-data, as that will allow your files to be interpreted properly.</p>
                <p><i>Sequencing experiment submitters</i>: We do not enforce strict formatting requirements on processed files, except for bam (alignment) files, which should follow <a href="/ena/about/read-file-formats#bam_format">bam file specification from the European Nucleotide Archive</a>. The alignments should also be generated against a reference genome which has been deposited in the International Nucleotide Sequence Database Collaboration (INSDC, involving <a href="http://www.ddbj.nig.ac.jp">DDBJ</a>, <a href="/ena">ENA</a>, and <a href="http://www.ncbi.nlm.nih.gov/genbank/">GenBank</a>). See this <a href = "UHTS_submissions.html#HowToSubmit">sequencing submissions help page</a> for more details.</p>
            </div>

            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>
            <p>&nbsp;</p>
            <p>&nbsp;</p>
            <h3><a id="ExptSubTools"></a>2. Which experiment submission tool to use?</h3>
            
            <p>Both tools are designed to help you create submissions which adhere to the MIAME and MINSEQE guidelines:</p>

            <table>

                <tr>
                    <th>Tool</th>
                    <th>Experiment type</th>
                    <th>Tool features</th>
                    <th>Help</th>
                </tr>
                <tr>
                    <td>
                        <h5 class="icon icon-functional" data-icon="_"><a href="/cgi-bin/microarray/magetab.cgi">MAGE-TAB spreadsheet tool</a></h5>
                    </td>
                    <td><ul>
                        <li>All experiments</li>
                    </ul>    
                    </td>
                    <td><ul>
                        <li>Meta-data spreadsheet template</li>
                        <li>Upload filled spreadsheet</li>
                        <li>Upload .zip or .gz data file bundles</li>
                    </ul>
                    </td>

                    <td><ul>
                        <li><a href="/fgpt/magetab/help/">MAGE-TAB help</a></li>
                        <li><a href="/training/online/course/arrayexpress-submitting-data-using-mage-tab">interactive tutorial</a></li>
                    </ul>
                    </td>

                </tr>
                <tr>
                    <td>
                        <h5 class="icon icon-functional" data-icon="_"><a href="/fg/annotare/login">Annotare webform tool</a></h5>
                    </td>
                    <td><ul>
                        <li>Intended for 15 samples or fewer with no complex design (e.g. pooling of samples)</li>
                        <li>Larger/complex experiments supported soon</li>
                    </ul>    
                    </td>
                    <td><ul>
                        <li>Fill in a series of webforms</li>
                        <li>Upload data files one by one</li>
                        <li>Validate prior to submission</li>
                    </ul>
                    
                    </td>
                    
                    <td><ul>
                        <li><a href="/fgpt/annotare_help/">Annotare help</a></li>                        
                    </ul>
                    </td>
                </tr>
            </table>


            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>

            <h3><a id="WhatToExpect"></a>3. What to expect from each experiment submission tool</h3>


            <h4>MAGE-TAB</h4>
            <p>You will be sending us a meta-data spreadsheet and data archives (e.g. .zip or .gz files) containing your data files.  Don't create the spreadsheet from scratch but use the <a href="/cgi-bin/microarray/magetab.cgi">MAGE-TAB submission tool</a> to generate a template tailored for your experiment. Always use a freshly-created template for a new submission because the template-generation system is updated regularly.</p>
            <p>Download the template spreadsheet file (which is a tab-delimited text file) to fill it in offline.  The file will open in any spreadsheet program, e.g. <a href="https://wiki.openoffice.org/wiki/Documentation/FAQ/Calc/Files/How_do_I_open_a_tab-delimited_file_in_OpenOffice.org_Spreadsheet%3F_What_if_I_have_a_different_type_of_delimiter%3F">OpenOffice Calc</a> or <a href="http://office.microsoft.com/en-gb/excel-help/import-or-export-text-txt-or-csv-files-HP010342598.aspx">Microsoft Excel</a>. Most of the row/column headings in the template are mandatory, so please DO NOT remove or edit them. Refer to this <a href="fgpt/magetab/help/">MAGE-TAB help page</a> for help with constructing the MAGE-TAB spreadsheet and data files.  Save the template as a tab-delimited text file for submission.</p>
           <p>Please note we will not be able to process any unfilled template spreadsheets or spreadsheets in Excel format (*.xls or *.xlsx files).</p>
           <p>To submit, login to the submission tool again and upload the spreadsheet along with your data files. Make sure you click "Submit Experiment" at the very end, after which you should receive an automatic confirmation email.  For those who are new to MAGE-TAB submissions, please check this <a href="/fgpt/magetab/help/quick_start.html">quick start guide</a>.</p>

            <h4>Annotare</h4>
            <p>With <a href="/fg/annotare/login">Annotare</a>, there will be a series of webforms with text boxes to fill, drop-down lists to select terms from, and also a data-file upload functionality inside the tool which allows you to upload files one by one and then assign each of them to specific hybridisation/sequencing assay.</p>
            <p>Before you submit the experiment to ArrayExpress, the information you've provided will be validated by Annotare.  Any validation errors (e.g. missing information) will be highlighted. Please fix the issues and validate again because only experiments which pass validation can be submitted. If the errors persist, please refer to this <a href="/fgpt/annotare_help/">Annotare help page</a> or contact the Annotare team at <a href="mailto:annotare@ebi.ac.uk">annotare@ebi.ac.uk</a>.</p>

            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>

            <h3><a id="CommonQuestions"></a>4. Common questions</h3>
            <ul>
                <li><strong>General submission questions</strong> (MAGE-TAB and MIAMExpress): <a href="faq.html#submit_faq_general">Submission FAQ</a></li>
                <li>How to <strong>prepare MAGE-TAB submission</strong>: <a href="/fgpt/magetab/help">MAGE-TAB help</a></li>
                <li>How to <strong>use Annotare</strong> for submission: <a href="/fgpt/annotare_help/">Annotare help</a></li>
                
                <li>Getting ArrayExpress <strong>accession number</strong> for your experiment: <a href="faq.html#need_acc_num_urgent">Accession number</a></li>
                <li>How to change <strong>release dates</strong> for your unpublished data: <a href="data_availability.html#changes">Release date changes</a></li>
            </ul>
            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>


            <h3><a id="ADFSub"></a>5. Array design submissions</h3>

            <h4>What is an array design?</h4>

            <p>For each microarray experiment submission, the submission tool will ask you for the array design (i.e. microarray chip/platform) used for each hybridisation. Many of these designs, especially commercially available ones, are already accessioned in ArrayExpress as array design format (ADF) files.  (One ADF is for one microarray chip.) The ADF file is a tabulated (spreadsheet-like) tab-delimited text file that contains information about all the probes on the microarray chip, e.g. how the probes were designed, the probe sequences, probe coordinate location (on spotted arrays), cross-references to other database, where the probes map to in the genome, etc.</p>

            <h4>Do I need to submit an array design?</h4>
            <p>If your ADF is already accessioned in ArrayExpress, you do <em>not</em> need to submit any ADF to us. All you need to do is to provide the ArrayExpress accession number of the ADF in the "Array Design REF" column of your MAGE-TAB meta-data spreadsheet.  To find out whether your array design is already in ArrayExpress and get its accession, please search by keywords (e.g. manufacturer name, species name) on the <a href="/arrayexpress/arrays/browse.html?directsub=on">ArrayExpress array design search page</a>. If the search does not return any results, please <a href="/arrayexpress/arrays/browse.html">broaden your search here</a> to include array designs we imported from <a href="http://www.ncbi.nlm.nih.gov/geo">NCBI Gene Expression Omnibus (GEO)</a> (accession numbers prefixed by "A-GEOD"). As a rule of thumb, a GEO platform accession can be converted to the ArrayExpress equivalent by replacing "GPL" with "A-GEOD", e.g. "GPL1234" will be "A-GEOD-1234". See this <a href="how_to_search.html#SearchADF">ADF search help page</a> for further details.</p>

            <h4>How do I submit commercially available array designs?</h4>
            <p>If you use a commercially available microarray which is not accessioned in ArrayExpress, please write to us at <a href="mailto:miamexpress@ebi.ac.uk">miamexpress@ebi.ac.uk</a>. In your email, please tell us which microarray was used and where the annotation/support files can be downloaded. An ArrayExpress curator should then be able to create the ADF and get it accessioned in ArrayExpress. </p>

            <h4>How do I submit custom array designs?</h4>
            <p>If you have a custom microarray which is not accessioned in ArrayExpress at all and is not commercially available, you will have to generate the array design format (ADF) file following <a href="/miamexpress/help/array_designs.html">these ADF construction guidelines</a>. Submit the ADF using the <a href="/miamexpress/login.html">MIAMExpress web interface tool</a>.</p>

            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>
        </section>
        <aside class="grid_6 omega">
            <h3>Help</h3>
            <ul id="secondary-nav">
                <li><a href="index.html">Help home</a></li>
                <li><a href="quick_start.html">Quick Start guide</a></li>
                <li><a href="how_to_search.html">How to search</a></li>
                <li><a href="programmatic_access.html">Other ways to get data</a>
                    <ul>
                        <li><a href="programmatic_access.html">Programmatic access</a></li>
						<li><a href="FTP_archives.html">FTP archives</a></li>
						<li><a href="expressionset_objects.html">ExpressionSet R objects</a></li>
                    </ul>
                </li>
                <li class="active"><a href="submissions_overview.html">How to submit</a>
                    <ul>
                        <li><a href="UHTS_submissions.html">Sequencing submissions</a></li>
                        <li><a href="third_party_tools.html">Third-party tools</a></li>
                    </ul>
                </li>
                <li><a href="FAQ.html">FAQ</a></li>
                <li><a href="data_availability.html">Learn more about our data</a>
                    <ul>
                        <li><a href="data_availability.html">Data access policy</a></li>
                        <li><a href="accession_codes.html">Accession codes</a></li>
                        <li><a href="GEO_data.html">GEO data</a></li>
                    </ul>
                </li>
                <li><a href="contact_us.html">Contact us</a></li>
            </ul>
        </aside>
    </div>
</body>
</html>
