<!DOCTYPE html>
<html class="no-js" lang="en">
<head><title>Submissions Overview &lt; ArrayExpress &lt; EMBL-EBI</title></head>
<body class="level2">
    <div id="content" role="main" class="grid_24 clearfix">
        <section id="ae-help" class="grid_18 alpha">
            <h2>How to submit</h2>
            <div class="grid_24">
                <div class="calloutbox" style="padding: 20px; position: relative; background-color: rgb(238, 249, 243);">
                    <h3 class="icon icon-functional" data-icon="_"><a href="/cgi-bin/microarray/magetab.cgi">MAGE-TAB spreadsheet tool</a></h3>
                    <br>
                    <ul>
                        <li>All sequencing and microarray experiment submissions</li>
                        <li><a href="/fgpt/magetab/help">MAGE-TAB submission help</a></li>
                        <li><a href="/training/online/course/arrayexpress-submitting-data-using-mage-tab">MAGE-TAB submission interactive tutorial</a></li>                       
                    </ul>
                </div>
            </div>

            <p>&nbsp;</p>
            <p><strong>MIAMExpress users:</strong> (16 October 2013) The MIAMExpress submission tool is no longer maintained for experiment submissions. If you're starting a new experiment submission, please use the MAGE-TAB spreadsheet tool (see above). Array design submissions are still accepted via MIAMExpress.</p>
            <p>&nbsp;</p>

            
            <h3>More information about submissions</h3>
            <p>
                <a href="#WhatIsASub">1. What do I need to submit?</a><br>
                <a href="#ExptSubTools">2. Which experiment submission tool to use?</a><br>
                <a href="#WhatToExpect">3. What to expect from each experiment submisssion tool</a><br>
                <a href="#CommonQuestions">4. Common questions</a><br>
                <a href="#ADFSub">5. Array Design submissions</a><br>
            </p>
            <h3><a id="WhatIsASub"></a>1. What do I need to submit?</h3>
            <p>ArrayExpress accepts all functional genomics data, including microarray-based and <a href="UHTS_submissions.html">sequencing</a> data. Microarray submissions follow the "Minimum Information About a Microarray Experiment" (<a href='http://www.mged.org/Workgroups/MIAME/miame_2.0.html'>MIAME</a>) guidelines, while sequencing submissions follow a similar set of guidelines, "Minimum Information About a Sequencing Experiment" (<a href='http://www.mged.org/minseqe/'>MINSEQE</a>).  Following the guidelines, for experiment submissions, we ask submitters to provide the following:</p>
            <div class="grid_1 alpha">
                <p>&nbsp;</p>
            </div>

            <div  class="grid_17 omega">
                <p><strong>Meta-data</strong>: E.g. Background biology of your experiment, aim of the experiment, the biological materials/samples used and their characteristics, wet-lab and dry-lab procedures (protocols).</p>
                <p><strong>Raw data files</strong>: These are unprocessed data files obtained from the microarray scanner (e.g. Affymetrix CEL files, Agilent feature extraction files) or from the sequencing machine (e.g. fastq files).</p>
                <p>Microarray experiment submitters: Please make sure the format of your raw data files matches one of the <a href="/fgpt/magetab/help/data_format.html#raw">accepted formats</a>.  If your experiment was performed with the Illumina BeadChip platform, make sure you generated raw data files using the <a href="http://www.illumina.com/software/genomestudio_software.ilmn">Illumina GenomeStudio software </a>(previously known as the "BeadStudio" software) with the 'ArrayExpress Data Submission Report Plug-in'. The plug-in is available from the <a href="http://www.illumina.com/software/illumina_connect.ilmn#gene_expression_plug_ins">Illumina connect website</a>. The raw data files should be formatted as in this <a href="/fgpt/magetab/help/data_format.html#illumina">BeadChip raw data example</a>.</p>
                <p>Sequencing experiment submitters: Please make sure your raw sequence read files are formatted and prepared according to <a href="/ena/about/sra_data_format">specifications by the European Nucleotide Archive (ENA)</a>.  Please refer to this <a href = "UHTS_submissions.html">specific help page on sequencing submissions</a> for more details.</p>
                <p><strong>Processed data files</strong>: These are data files generated from the raw data files, often involving normalisation, filtering, background-subtraction. In sequencing expermient, the processing can also involve alignment of sequencing reads to a reference genome, calculation of RPKM/FPKM values, etc.</p>
                <p>Microarray experiment submitters: If you have one processed data file for each hybridisation assay, we recommend that the file to be formatted as described on this <a href="/fgpt/magetab/help/data_format.html#normalized">per-hybridisation normalised file example</a>. If you are submitting a single processed file containing data from multiple hybridisation assays, depending on the submission tool used, we recommend the <a href="http://tab2mage.sourceforge.net/docs/datafiles.html#final_data_matrix">data matrix format (MIAMExpress tool)</a> or <a href="/fgpt/magetab/help/data_format.html#data_matrix">data matrix format (MAGE-TAB submission tool)</a>. We no longer enforce strict formatting requirements for processed data files but you are encouraged to use the recommended formats. Regardless of the type and format of processed data you are sending us, please make sure you have provided a detailed normalisation/data transformation protocol in your meta-data, as that will allow your data to be interpreted properly.</p>
                <p>Sequencing experiment submitters: We do not enforce strict formatting requirements on processed files, except for bam (alignment) files, which should follow <a href="http://www.ebi.ac.uk/ena/about/sra_data_format#bam_format">bam file specification from the European Nucleotide Archive</a>. The alignments should also be generated against a reference genome which has been deposited in the International Nucleotide Sequence Database Collaboration (INSDC, involving <a href="http://www.ddbj.nig.ac.jp">DDBJ</a>, <a href="/ena">ENA</a>, and <a href="http://www.ncbi.nlm.nih.gov/genbank/">GenBank</a>). See this <a href = "UHTS_submissions.html#HowToSubmit">sequencing submissions help page</a> for more details.</p>
            </div>

            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>
            <p>&nbsp;</p>
            <p>&nbsp;</p>
            <h3><a id="ExptSubTools"></a>2. Which experiment submission tool to use?</h3>
            <h5>We provide two tools, "MAGE-TAB" and "MIAMExpress". How do they differ?</h5>
            <p>Both tools are designed to help you create submissions which adhere to the MIAME and MINSEQE guidelines. The spreadsheet-based "MAGE-TAB" tool is more versatile and can handle all types of submissions, while the web-form tool "MIAMExpress" has a limited scope and is no longer maintained (kept here only for submitters who have already started a submission with the tool):</p>


            <table>
                <tr>
                    <th>Tool</th>
                    <th>Size of experiment</th>
                    <th>Type of data</th>
                    <th>Help</th>
                </tr>
                <tr>
                    <td>
                        <h5 class="icon icon-functional" data-icon="_"><a href="/cgi-bin/microarray/magetab.cgi">MAGE-TAB spreadsheet tool</a></h5> - recommended
                    </td>
                    <td>any size</td>
                    <td>All experiments, including microarray and sequencing data. <a href="UHTS_submissions.html">Sequencing experiments</a> and <a href="faq.html#illumina"> Illumina BeadChip experiments</a> must be submitted with this tool</td>
                    <td><ul>
                        <li><a href="/fgpt/magetab/help/">MAGE-TAB help</a></li>
                        <li><a href="/training/online/course/arrayexpress-submitting-data-using-mage-tab">interactive tutorial</a></li>
                        <li><a href="faq.html#illumina">Illumina help</a></li>
                        <li><a href="UHTS_submissions.html">sequencing help</a></li>
                    </ul>
                    </td>

                </tr>
                <tr>
                    <td>
                        <h5 class="icon icon-functional" data-icon="_"><a href="/miamexpress/login.html">MIAMExpress web interface tool (no longer maintained)</a></h5>
                    </td>
                    <td>&lt; 15 hybs</td>
                    <td>
                    <strong>If you're starting a new experiment submission, please use the MAGE-TAB spreadsheet tool (see above).</strong><br>Non-sequencing, non-Illumina microarray experiments with 1 raw data file per hybridization, no dye-swaps<p></p></td>
                    <td><ul>
                        <li><a href="/miamexpress/help/index.html">overview</a></li>
                        <li><a href="/miamexpress/help/web_ui.html">webform help</a></li>
                    </ul>
                    </td>
                </tr>
            </table>


            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>

            <h3><a id="WhatToExpect"></a>3. What to expect from each experiment submission tool</h3>


            <h4>MAGE-TAB</h4>
            <p>You will be submitting to us a meta-data spreadsheet and data archives (e.g. .zip or .gz files) containing your raw and/or processed data files.  You don't have to create the spreadsheet from scratch: use the <a href="/cgi-bin/microarray/magetab.cgi">MAGE-TAB submission tool</a> to generate a template spreadsheet tailored for your experiment. The template-generation system is updated regularly so it's a good practice to create and use a new template spreadsheet for each new submission, <i>even if you have submitted to us before</i>.</p>
            <p>Download the template spreadsheet to work offline in your own time. Most of the row/column headings in the template are mandatory and populated by terms from a controlled vocabulary, hence should not be removed or edited. Please take care when filling in the template spreadsheet with details of your experiment and samples. Refer to this <a href="fgpt/magetab/help/">MAGE-TAB help page</a> for help with constructing the MAGE-TAB spreadsheet and data files.</p>
           <p>When you're ready to submit, login to the MAGE-TAB submission tool again and upload the spreadsheet along with your data files. If you are new to MAGE-TAB submissions, please have a look at this <a href="/fgpt/magetab/help/quick_start.html">quick start guide</a>.</p>

            <h4>MIAMExpress</h4>
            <p><strong>This tool is no longer maintained for experiment submissions. Please use MAGE-TAB spreadsheet tool if you're submitting a new experiment.</strong></p><p>You will be filling in meta-data information on the <a href="/miamexpress/login.html">MIAMExpress web interface tool</a>. There are two ways to use the tool: batchloader mode (recommended) or web-form mode. Links to both methods are found within the 'experiment submission' section of the MIAMExpress tool.  In both the pages are ordered in a way to reflect the chronological flow of your experiment, e.g. you will first be asked about the source materials, then the RNA/DNA extracts, the labelled extracts and hybridisations. The form will ask you to upload raw/processed data files only <em>after</em> you have described the "hybridisations" sufficiently in your experiment (e.g. which microarray design/platform was used for the hybridisation, which labelled extract went into the hybridisation, etc).</p>

            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>

            <h3><a id="CommonQuestions"></a>4. Common questions</h3>
            <ul>
                <li><strong>General submission questions</strong> (MAGE-TAB and MIAMExpress): <a href="faq.html#submit_faq_general">Submission FAQ</a></li>
                <li>How to <strong>prepare MAGE-TAB files </strong> for submission: <a href="/fgpt/magetab/help">MAGE-TAB help</a></li>
                
                <li>Getting ArrayExpress <strong>accession number</strong> for your experiment: <a href="faq.html#need_acc_num_urgent">Accession number</a></li>
                <li>How to set/change <strong>release dates</strong> for your data: <a href="data_availability.html">ArrayExpress data access policy</a></li>
                <li><strong>MIAMExpress-specific</strong> submission questions: <a href="faq.html#Submitter_FAQ_mx">MIAMExpress FAQ</a> (Please note that MIAMExpress is no longer maintained for new experiment submissions. Array design sbumissions are still accepted via MIAMExpress.)</li>
                
                <li>All other questions: please try <a href="faq.html">General FAQ</a> or email us at <a href="mailto:miamexpress@ebi.ac.uk">miamexpress@ebi.ac.uk</a></li>
            </ul>
            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>


            <h3><a id="ADFSub"></a>5. Array design submissions</h3>

            <h4>What is an array design?</h4>

            <p>For each microarray experiment submission, the submission tool will ask you for the array design (i.e. microarray chip/platform) used for each hybridisation. Many of these designs, especially commercially available ones, are already accessioned in ArrayExpress as array design format (ADF) files.  (One ADF is for one microarray chip.) The ADF file is a tabulated (spreadsheet-like) tab-delimited text file that contains information about all the probes on the microarray chip, e.g. how the probes were designed, the probe sequences, probe coordinate location (on spotted arrays), cross-references to other database, where the probes map to in the genome, etc.</p>

            <h4>Do I need to submit an array design?</h4>
            <p>If your ADF is already accessioned in ArrayExpress, you do <em>not</em> need to submit any ADF to us. All you need to do is to provide the ArrayExpress accession number of the ADF in the "Array Design REF" column of your MAGE-TAB meta-data spreadsheet.  To find out whether your array design is already in ArrayExpress and get its accession, please search by keywords (e.g. manufacturer name, species name) on the <a href="/arrayexpress/arrays/browse.html?directsub=on">ArrayExpress array design search page</a>. If the search does not return any results, please <a href="/arrayexpress/arrays/browse.html">broaden your search here</a> to include array designs we imported from <a href="http://www.ncbi.nlm.nih.gov/geo">NCBI Gene Expression Omnibus (GEO)</a> (accession numbers prefixed by "A-GEOD"). As a rule of thumb, a GEO platform accession can be converted to the ArrayExpress equivalent by replacing "GPL" with "A-GEOD", e.g. "GPL1234" will be "A-GEOD-1234".</p>

            <h4>How do I submit commercially available array designs?</h4>
            <p>If you use a commercially available microarray which is not accessioned in ArrayExpress, please write to us at <a href="mailto:miamexpress@ebi.ac.uk">miamexpress@ebi.ac.uk</a>. In your email, please tell us which microarray was used and where the annotation/support files can be downloaded. An ArrayExpress curator should then be able to create the ADF and get it accessioned in ArrayExpress. </p>

            <h4>How do I submit custom array designs?</h4>
            <p>If you have a custom microarray which is not accessioned in ArrayExpress at all and is not commercially available, you will have to generate the array design format (ADF) file following <a href="/miamexpress/help/array_designs.html">these ADF construction guidelines</a>. Submit the ADF using the <a href="/miamexpress/login.html">MIAMExpress web interface tool</a>. (The tool is still maintained for ADF submissions.)</p>

            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>
        </section>
        <aside class="grid_6 omega">
            <h3>Help</h3>
            <ul id="secondary-nav">
                <li><a href="index.html">Help home</a></li>
                <li><a href="quick_start.html">Quick Start guide</a></li>
                <li><a href="how_to_search.html">How to search</a></li>
                <li><a href="programmatic_access.html">Other ways to get data</a>
                    <ul>
                        <li><a href="programmatic_access.html">Programmatic access</a></li>
						<li><a href="FTP_archives.html">FTP archives</a></li>
						<li><a href="expressionset_objects.html">ExpressionSet R objects</a></li>
                    </ul>
                </li>
                <li class="active"><a href="submissions_overview.html">How to submit</a>
                    <ul>
                        <li><a href="UHTS_submissions.html">Sequencing submissions</a></li>
                        <li><a href="third_party_tools.html">Third-party tools</a></li>
                    </ul>
                </li>
                <li><a href="FAQ.html">FAQ</a></li>
                <li><a href="data_availability.html">Learn more about our data</a>
                    <ul>
                        <li><a href="data_availability.html">Data access policy</a></li>
                        <li><a href="accession_codes.html">Accession codes</a></li>
                        <li><a href="GEO_data.html">GEO data</a></li>
                    </ul>
                </li>
                <li><a href="contact_us.html">Contact us</a></li>
            </ul>
        </aside>
    </div>
</body>
</html>
