<!DOCTYPE html>
<html class="no-js" lang="en">
<head><title>Sequencing Submissions &lt; Help &lt; ArrayExpress &lt; EMBL-EBI</title></head>
<body class="level2">
    <div id="content" role="main" class="grid_24 clearfix">
        <section class="grid_18 alpha">
            <h2>Sequencing Submissions</h2>
            <p>
                <a href="#DataTypes">1. Types of data that can be submitted</a><br>
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#HumanIdentifiableData"> Potentially identifiable human data</a><br>
                <a href="#HowToSubmit">2. What data files to submit, and how</a><br>
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#meta_data">Meta-data</a><br>
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#raw_data_files">Raw data files</a><br>
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#MD5">MD5 checksum of raw files</a><br>
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#sending_raw_files">Sending raw files</a><br>
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#processed_data">Processed data files</a><br>
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#mixed_tech">Microarray and sequencing experiments (mixed technology)</a><br>
                <a href="#BamSpec">3. BAM file requirements</a><br>
                <a href="#Examples">4. Example submissions</a><br>
            </p>
            <br>
            <h3><a id="DataTypes"></a>1. Types of data that can be submitted</h3>
            <p>ArrayExpress accepts submissions of functional genomics data generated using high throughput sequencing (HTS) assays like RNA-seq and ChIP-seq, mostly from <strong>non-human and human non-identifiable</strong> samples. (If you're submitting potentially identifiable human data, please <a href="#HumanIdentifiableData">see below</a>.)</p>
            <p>To submit to ArrayExpress, all you need to do is send us meta-data for your experiment (e.g. experiment description, samples and their attributes, all protocols used) and the raw data files. Submissions without raw data files will not be accepted. See the <a href="#HowToSubmit">submission guide below</a>.</p>
            <p>The meta-data about your experiment will be stored at ArrayExpress, and the raw data files (e.g. fastq files) are eventually stored at the <a href="/ena/about/sra_submissions">Sequence Read Archive</a> (SRA) of the <a href="/ena/">European Nucleotide Archive</a> (ENA). ArrayExpress will transfer the raw data files to the ENA for you so you <i>do not</i> need to submit those files separately to the ENA. You can also send us processed data (i.e. processed from the raw reads, e.g. BAM alignment files, differential expression data, expression values linked to genome coordinates, etc).  Depending on the file format, it will either be stored at ArrayExpress or the ENA.</p>
            <p><a id="HumanIdentifiableData"></a><strong>Potentially identifiable human data</strong>:  Data from human samples and individuals that can potentially lead to the identification of the donors (e.g. genomic DNA sequences) can be submitted to ArrayExpress if the data has been consented for public release. Such approvals typically would be given by the relevant ethics committees and <em>ensuring this is the responsibility of the submitters</em>.</p>
            <p>Identifiable data approved for <a href="/ega/submission/FAQ#DAC">controlled access</a> should be submitted to the <a href="/ega/">European Genome-phenome Archive (EGA)</a>. Cases are possible where identifiable data (e.g. raw sequences) are submitted to the EGA, while the related processed data (e.g. RPKM values) are submitted to ArrayExpress, but <em>it is up to the submitter to ensure that such a submission copies to the respective ethics requirements.</em>  To submit processed data to ArrayExpress, please begin by emailing us at <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a> with the EGA study accession number. We will import non-human-identifiable meta-data from EGA in a spreadsheet (which the submitter will have the chance to review), and then match the meta-data with processed data.</p>
            <p>The following diagram summarises the typical data flow:.</p>
            <p><img src="UHTS_HighThroughputSequencingSubmissionDiagram.jpg" alt="Diagram of data submission routes" height="328" width="542"></p>
            <p>Diagram of the submission of different sequencing data types to ArrayExpress or the European Genome-phenome archive.</p>
            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>
            <p>&nbsp;</p>
                     
            <h3><a id="HowToSubmit"></a>2. What data files to submit, and how</h3>
            <p><a id="meta_data"></a><strong>Experiment meta-data</strong>: It should be submitted via the <a href="/fg/annotare/login">Annotare webform submission tool</a> (further described on this <a href="submissions_overview.html">submissions overview page</a>). Please note that our older  <a href="/cgi-bin/microarray/magetab.cgi">MAGE-TAB spreadsheet submission tool</a> is to be retired by 31 August 2014 and is only available for submitters to finish off existing submissions.  If for some reasons you cannot use Annotare, please contact us at <a href="mailto:annotare@ebi.ac.uk">araryexpress@ebi.ac.uk</a> and we will advise further.</p>
            
            <p><a id="raw_data_files"></a><strong>Raw data files</strong>: Please prepare them according to <a href='/ena/about/read-file-formats'>ENA specifications</a> (e.g. each individual fastq file should be compressed by gzip or bzip2). Data files which do not satisfy ENA's requirements will not be accepted. This is a developing field so please do check the specifications every time you submit a new experiment. If you are submitting BAM files as raw data files, please read this important documentation on <a href="#BamSpec">BAM file specification</a>.</p>
            
            <p><a id="MD5"></a><strong>MD5 checksum of raw files</strong>: As sequence files tend to be very large, we implement a file integrity validation step before sending them to the ENA on your behalf. For each sequence file that you transfer to ArrayExpress, please calculate its MD5 checksum and enter the value (without any whitespaces) in the MAGE-TAB spreadsheet or on the Annotare tool.  The checksum is hexadecimal and expressed as a long string of letters and numbers, which looks something like this: <code>eef75461035fb66d9173799d4e26ea97</code>. MD5 checksums are like the files' digital "fingerprints" and it is very unlikely that two non-identical files would generate the same "fingerprint", thus allowing us to verify that each file has not been corrupted during FTP transfer. Remember to calculate the checksum from the actual compressed file (e.g. fastq.gz or fastq.bz2) that is sent to us, not from the uncompressed fastq file or a re-compressed version created at a later time.</p>

            <p>How to calculate MD5 checksums: <a href="http://www.winmd5.com/">Windows user example</a>, <a href="http://www.macappguide.com/2009/04/28/quick-tip-check-md5sum-in-os-x/">Mac user example</a>, <a href="http://heatware.net/linux-unix/how-to-create-md5-checksums-and-validate-a-file-in-linux/">Linux user example</a>.</p> 
            
            <p><a id="sending_raw_files"></a><strong>Sending raw files:</strong> You must send the raw files to ArrayExpress by FTP (see <a href="ftp_upload.html">FTP transfer instructions</a>). Please transfer the compressed files one by one (and not bundling multiple fastq.gz files in one tar.gz archive) to avoid time-out issues and to allow us to process your files promptly. If you're an <a href="/fg/annotare/login">Annotare</a> submitter, go to the "FTP" file upload tab on the "Sample and Data" page, and follow on-screen instructions to fill in file names and their corresponding MD5 checksums.  If you're submitting with the <a href="/cgi-bin/microarray/magetab.cgi">MAGE-TAB tool</a>, please email us at <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a> to tell us about the files so that we can retrieve them from the FTP site.</p>
            
            <p><a id="processed_data"></a><strong>Processed data files</strong>: For small (under 5Mb) files, if you're using the <a href="/fg/annotare/login">Annotare</a> submission tool, they do not require compression and can be uploaded directly. If you're using the <a href="/cgi-bin/microarray/magetab.cgi">MAGE-TAB submission tool</a>, small files should be compressed into .zip or .gz archives and uploaded alongside your MAGE-TAB spreadsheet. For any files larger than 5MB, please send them by <a href="ftp_upload.html">FTP</a>. <a href="/fg/annotare/login">Annotare</a> users should go to the "FTP" file upload tab on the "Sample and Data" page and fill in processed file names and their corresponding MD5 checksums.  <a href="/cgi-bin/microarray/magetab.cgi">MAGE-TAB submission tool</a> users should email us at <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a> when the transfer has finished.</p>
            
            <p><a id="mixed_tech"></a><strong>If you have both sequencing and microarray data related to the same study</strong>, using the <a href="/cgi-bin/microarray/magetab.cgi">MAGE-TAB submission tool</a>, you can either submit them as one single ArrayExpress experiment (especially if you used the same biological samples for both data sets), or separate ArrayExpress experiments:</p>
            <ul>
                <li>When creating a single submission, you will have one "IDF" (Investigation description format) file describing your study, contact details and all wet-lab/dry-lab protocols used, which is then linked to multiple "SDRFs" (Sample and Data Relationship format) files describing the samples used and the data files they're associated with, e.g. one SDRF for sequencing data and another SDRF for microarray data.  See experiment <a href="/experiments/E-GEOD-32120">E-GEOD-32120</a> as an example.</li>
                <li>If you would prefer to submit your data as separate ArrayExpress experiments, that is fine too. Just email us at <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a> and let us know that your submissions are related, and we will cross-reference them in their description fields (e.g. "Data from an accompanying microarray experiment has also been deposited at ArrayExpress under accession xxxx"). See experiments <a href="/arrayexpress/experiments/E-MTAB-1432">E-MTAB-1432</a> and <a href="/arrayexpress/experiments/E-MTAB-1433">E-MTAB-1433</a> as examples.</li>
            </ul>
            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>
            <p>&nbsp;</p>
            <h3><a id="BamSpec"></a>3. BAM data file requirements</h3>
            <p>If you're submitting raw data files in BAM format, please make sure they satisfy <a href="/ena/about/read-file-formats#bam_format">ENA  specifications</a> as well as the following two conditions:</p>
            <ol><li>Each file contains all reads from the sequencing machine, regardless of whether the reads mapped to the reference genome. The reason for this is that we would expect the BAM file to be used to regenerate <i>all</i> the sequencing reads, provided we are armed with information about the reference genome sequence used for generating the alignment in the first place.</li>
            <li>The alignment in the BAM file was generated against a reference genome accessioned in the International Nucleotide Sequence Database Collaboration (INSDC, involving <a href="http://www.ddbj.nig.ac.jp">DDBJ</a>, <a href="/ena">ENA</a>, and <a href="http://www.ncbi.nlm.nih.gov/genbank/">GenBank</a>).</li></ol>
            <p>If your BAM files contain only mapped reads, then please either create "full", unfiltered BAM files (where both mapped and unmapped reads are present), or send us the original read files as raw data files (again, check <a href='/ena/about/read-file-formats'>ENA specifications</a>).</p>
            <p>BAM files containing only mapped reads can be included in your submission as processed files, as long as they satisfy ENA's specification and that the reference genome used for alignment has been accessioned in INSDC.</p>
            <p>&nbsp;</p>
            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>
            <h3><a id="Examples"></a>4. Example submissions</h3>
            <p>All examples are taken from public experiments on the <a href="/arrayexpress">ArrayExpress website</a>. For each of the examples below, there is a link to the experiment's frontpage, where you will find all the meta-data and curated spreadsheet (idf.txt / sdrf.txt) in the "Files" section to download. The sample-date relationship table, when read from left to right, shows you how the sample attributes are described and experiment workflow (from samples --> sequencing libraries --> sequencing assay --> data files) is captured.</p>
            <h4>RNA-seq, Illumina Genome Analyzer IIx platform, single-end</h4>
            <p><a href="/arrayexpress/experiments/E-MTAB-997">E-MTAB-997 experiment page</a>,
               <a href="/arrayexpress/experiments/E-MTAB-997/samples/?full=true">Sample-data relationship table</a></p>
            <h4>RNA-seq example, Illumina Genome Analyzer II platform, paired-end</h4>
            <p><a href="/arrayexpress/experiments/E-MTAB-1091">E-MTAB-1091 experiment page</a>,
               <a href="/arrayexpress/experiments/E-MTAB-1091/samples/?full=true">Sample-data relationship table</a></p>
            <h4>5C and HiC example, Illumina Genome Analyzer II platform, paired-end</h4>
            <p><a href="/arrayexpress/experiments/E-MTAB-1948">E-MTAB-1948 experiment page</a>,
               <a href="/arrayexpress/experiments/E-MTAB-1948/samples/?full=true">Sample-data relationship table</a></p>               
            <h4>ChIP-seq, AB SOLiD System, single-end</h4>
            <p><a href="/arrayexpress/experiments/E-MTAB-830">E-MTAB-830 experiment page</a>,
               <a href="/arrayexpress/experiments/E-MTAB-830/samples/?full=true">Sample-data relationship table</a></p>
            <h4>DNA-seq, AB SOLiD System 3.0 platform, paired-end</h4>
            <p><a href="/arrayexpress/experiments/E-MTAB-1082">E-MTAB-1082 experiment page</a>,
               <a href="/arrayexpress/experiments/E-MTAB-1082/samples/?full=true">Sample-data relationship table</a></p>
            <h4>ChIP-seq, Illumina HiSeq 2000 platform, single-end</h4>
            <p><a href="/arrayexpress/experiments/E-MTAB-1084">E-MTAB-1084 experiment page</a>,
               <a href="/arrayexpress/experiments/E-MTAB-1084/samples/?full=true">Sample-data relationship table</a></p>
            <p>&nbsp;</p>
            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>

        </section>
        <aside class="grid_6 omega">
            <h3>Help</h3>
            <ul id="secondary-nav">
                <li><a href="index.html">Help home</a></li>
                <li><a href="quick_start.html">Quick Start guide</a></li>
                <li><a href="how_to_search.html">How to search</a></li>
                <li><a href="programmatic_access.html">Other ways to get data</a>
                    <ul>
                        <li><a href="programmatic_access.html">Programmatic access</a></li>
						<li><a href="FTP_archives.html">FTP archives</a></li>
						<li><a href="expressionset_objects.html">ExpressionSet R objects</a></li>
                    </ul>
                </li>
                <li><a href="submissions_overview.html">How to submit</a>
                    <ul>
                        <li class="active"><a href="UHTS_submissions.html">Sequencing submissions</a></li>
                        <li><a href="third_party_tools.html">3rd party tools</a></li>
                    </ul>
                </li>
                <li><a href="FAQ.html">FAQ</a></li>
                <li><a href="data_availability.html">Learn more about our data</a>
                    <ul>
                        <li><a href="data_availability.html">Data access policy</a></li>
                        <li><a href="accession_codes.html">Accession codes</a></li>
                        <li><a href="GEO_data.html">GEO data</a></li>
                    </ul>
                </li>
                <li><a href="contact_us.html">Contact us</a></li>
            </ul>
        </aside>
    </div>
</body></html>
